kcqrs-core/build.gradle

```diff
@@ -26,6 +26,9 @@ configurations {
 
 dependencies {
   compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+  compile group: 'com.google.code.gson', name: 'gson', version: '2.7'
+
+  compile group: 'com.esotericsoftware', name: 'kryo', version: '4.0.0'
```

Comment:
Not able to see where kryo is used. I'm not sure that the core module should depend on this ones. We already have gson module, so if kryo is appropriate for this it would be better to be added as separate module.

Reply:
And adding a new methods for serialization and deserialization ( so that the old ones would not be affected) using a Gson with the setting to use annotations while serializing and deserializing.

Reply:
I'm a bit afraid of such usage (Direct Serialisation of AggregateRoot) as everyone could use it's own modifiers and access to the object which may cause and issues. Enforcing of such thing will require adding of tests for these serializations which are targeting these objects.